### PR TITLE
Theme - Onlywsx

### PR DESCRIPTION
**Pull request by onlywsx**
a new theme for sublime text

### CODE REVIEW

File: repository/t.json [hunk 1]
```diff
@@ -1217,6 +1217,17 @@
 			]
 		},
 		{
+			"name": "Theme - Onlywsx",
+			"details": "https://github.com/onlywsx/Theme---Onlywsx",
+			"labels": ["theme"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"branch": "master"
```
Comment (wbond) [inline repository/t.json]: Please see https://packagecontrol.io/docs/submitting_a_package#Step_4

**Comment by wbond** [comment]
Your readme references the Soda theme. You should remove that.

If you are using parts of the Soda theme, you need to follow the author's license.

**Comment by wbond** [comment]
The version number needs to be a semver, i.e. have thee numeric parts. In other words, `v1.0.0` is ok, but the current value of `v1.0` is not.

**Comment by onlywsx** [comment]
The current version number is v1.0.0

**Comment by FichteFoll** [comment]
>If you are using parts of the Soda theme, you need to follow the author's license.

Your package is clearly a derivative of the soda theme. Again, please follow the terms of the author's license.

**Comment by onlywsx** [comment]
already followed the terms of the Soda Theme's license.

**Comment by FichteFoll** [comment]
It is fine now. It wasn't the case when I made that comment, however.

**Pull request merged**